modules/apps/powwow/powwow-portlet/docroot/WEB-INF/src/com/liferay/powwow/service/impl/PowwowServerLocalServiceImpl.java

```diff
@@ -53,7 +53,6 @@ public PowwowServer addPowwowServer(
 		powwowServer.setModifiedDate(serviceContext.getModifiedDate(now));
 		powwowServer.setName(name);
 		powwowServer.setProviderType(providerType);
```

Comment:
how can `powwowServer.setProviderType(providerType)` still exists? we already remove it from `server.xml`, so when building service, this method no longer available and will raise compile error. do you think so? please double check other places that still relates to `providerType`.